flake8_variables_names/checker.py

```diff
@@ -67,13 +69,13 @@ def get_varname_errors(self, var_name: str, var_ast_node) -> List[Tuple[int, int
             errors.append((
                 var_ast_node.lineno,
                 var_ast_node.col_offset,
-                'VNE001 single letter variable names are not allowed',
+                'VNE001 single letter variable names like \'%s\' are not allowed' % (var_name,),
             ))
         if var_name in self.variable_names_blacklist:
             errors.append((
                 var_ast_node.lineno,
                 var_ast_node.col_offset,
-                'VNE002 variable name should be clarified',
+                'VNE002 variable name \'%s\' should be clarified' % (var_name,),
```

Comment:
Same thing with quotes here.

Reply:
Also use `.format` to comply style guide (link can be found in readme).

Reply:
Fixed. Also fix docstrig.

It is not clear for me from your styleguide, if single string formatting param requires numbering too. I presumed that it does.